apps/openssl.c

```diff
@@ -787,18 +793,23 @@ static void list_disabled(void)
 
 static LHASH_OF(FUNCTION) *prog_init(void)
 {
-    LHASH_OF(FUNCTION) *ret;
+    static LHASH_OF(FUNCTION) *ret = NULL;
+    static int prog_inited = 0;
     FUNCTION *f;
     size_t i;
 
-    /* Sort alphabetically within category. For nicer help displays. */
-    for (i = 0, f = functions; f->name != NULL; ++f, ++i) ;
-    qsort(functions, i, sizeof(*functions), SortFnByName);
+    if (!prog_inited) {
+        prog_inited = 1;
+
+        /* Sort alphabetically within category. For nicer help displays. */
+        for (i = 0, f = functions; f->name != NULL; ++f, ++i) ;
```

Comment:
please use continue.  or at least put the last semi-colon on its own line.

Reply:
Okie